src/Symfony/Component/ObjectMapper/Transform/MapCollection.php

```diff
@@ -41,4 +42,10 @@ public function __invoke(mixed $value, object $source, ?object $target): mixed
 
         return $values;
     }
+
+    public function withObjectMapper(ObjectMapperInterface $objectMapper): static
```

Comment:
wither method should return a clone - the method should be named setObjectMapper if we really want a setter